_topic_maps/_topic_map.yml

```
@@ -1917,6 +1917,8 @@ Topics:
     File: uninstalling-openshift-gitops
   - Name: Setting up a new Argo CD instance
     File: setting-up-argocd-instance
+    Name: Monitoring Argo CD instances
```

```suggestion
  - Name: Monitoring Argo CD instances
```
Add a hyphen and a space before `Name` (as in the other `Name` entries). Then push the changes and see if that fixes build issue.